training/vllm/Containerfile

```diff
@@ -0,0 +1,6 @@
+FROM quay.io/wxpe/tgis-vllm:release.4e3ff78
+
+USER root
+RUN ln -s /usr/lib64/libcuda.so.1 /usr/lib64/libcuda.so
+COPY mixtral.jinja .
```

Comment:
I would prefer the Model to be on the host (bootc container) rather then inside of one particular container this would give us the ability to share it with other containers potentially.

Reply:
this is just a jinja template, probably just needed for vllm as this is used to serve the model.